src/ports/linux/pnal_udp.c

```diff
@@ -40,14 +37,21 @@ int pnal_udp_open (pnal_ipaddr_t addr, pnal_ipport_t port)
          .sin_port = htons (port),
          .sin_zero = {0},
       };
-      ret = bind (id, (struct sockaddr *)&local, sizeof (local));
-      if (ret != 0)
+
+      if (setsockopt(id, SOL_SOCKET, SO_REUSEADDR, &enable, sizeof(enable)) < 0)
```

Comment:
!= 0

Reading the refman setsockopt() should return 0 on success. 
Any reason for not using same condition as for bind()?

Reply:
Done